screen-orientation/hidden_document.html

```diff
@@ -14,48 +14,22 @@
 
   promise_test(async (t) => {
     const { minimize, restore } = window_state_context(t);
+    t.add_cleanup(restore);
```

Comment:
`window_state_context` adds `resotre()` as clean up method. This is unnecessary. See https://github.com/web-platform-tests/wpt/pull/38031
```javascript
function window_state_context(t) {
    let rect = null;
    let state = "restored";
    t.add_cleanup(async () => {
        if (state === "minimized")
            await restore();
    });
```
I guess that you want to call `orientation.unlcok` as clean up. You should add `t.add_cleanup(makeCleanup());` instead.

Reply:
Ah, thanks for checking that.